eng/dependabot/Versions.props

```diff
@@ -0,0 +1,29 @@
+<Project>
+  <!-- Import references updated by Dependabot. This file is for package references updated manually or by Maestro. -->
+
+  <PropertyGroup>
+    <AzureIdentityVersion>1.6.0</AzureIdentityVersion>
+    <AzureStorageBlobsVersion>12.13.0</AzureStorageBlobsVersion>
+    <AzureStorageQueuesVersion>12.11.0</AzureStorageQueuesVersion>
+    <MicrosoftAspNetCoreAuthenticationJwtBearerVersionNet6>6.0.5</MicrosoftAspNetCoreAuthenticationJwtBearerVersionNet6>
```

Comment:
These four versions should be removed since dependabot won't understand what they are. Seems like they are unused because they still exist in `./eng/Versions.props`